.github/scripts/deploy_api.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+echo "Proxygen path: $PROXYGEN_PATH"
+echo "Specification path: $SPEC_PATH"
+echo "Stack name: $STACK_NAME" # instance
+echo "Target environment: $TARGET_ENVIRONMENT"
+
+if [ "$TARGET_ENVIRONMENT" != "dev-pr" ]; then
+  environment=$TARGET_ENVIRONMENT
+else
+  environment=internal-dev
+fi
+
+echo "Environment name: $environment"
+
+# Find and replace the x-nhsd-apim.target.url value
+jq --arg stack_name "$STACK_NAME" '.["x-nhsd-apim"].target.url = "https://\($stack_name).dev.eps.national.nhs.uk"' "$SPEC_PATH" > temp.json && mv temp.json "$SPEC_PATH"
+
+proxygen_private_key_arn=$(aws cloudformation list-exports --query "Exports[?Name=='account-resources:ProxgenPrivateKey'].Value" --output text)
+# proxygen_public_key_arn=$(aws cloudformation list-exports --query "Exports[?Name=='account-resources:ProxgenPublicKey'].Value" --output text)
+
+proxygen_private_key=$(aws secretsmanager get-secret-value --secret-id "${proxygen_private_key_arn}" --query SecretString --output text)
+# proxygen_public_key=$(aws secretsmanager get-secret-value --secret-id "${proxygen_public_key_arn}" --query SecretString --output text)
+
+# client_private_key_arn=$(aws cloudformation list-exports --query "Exports[?Name=='account-resources:PsuClientKeySecret'].Value" --output text)
+# client_public_key_arn=$(aws cloudformation list-exports --query "Exports[?Name=='account-resources:PsuClientCertSecret'].Value" --output text)
+
+# client_private_key=$(aws secretsmanager get-secret-value --secret-id "${client_private_key_arn}" --query SecretString --output text)
+# client_public_key=$(aws secretsmanager get-secret-value --secret-id "${client_public_key_arn}" --query SecretString --output text)
```

Comment:
can you remove commented out code if its not used

Reply:
Commented out code has been removed.

Reply:
I have realized why they are there - we need some commands further down before we do a deploy to set the secrets for mutual tls using proxygen

Reply:
Putting secret psu-mtls-2 in internal-dev
```
{
  "name": "psu-mtls-2",
  "last_modified": "2024-04-16T14:11:10.420000+00:00",
  "environment": "internal-dev",
  "version_id": "be3c5980-5bce-40c6-a360-445c3107cb0a",
  "type": "mtls",
  "in_use_by": [
    "prescription-status-update-api--internal-dev--psu-pr-99-sandbox"
  ]
}
```